src/openrct2/management/NewsItem.h

```diff
@@ -52,14 +52,63 @@ struct NewsItem
     uint16_t MonthYear;
     uint8_t Day;
     utf8 Text[256];
```

Comment:
Not sure whether you would like to include the extra work in the PR or not, if so then this should become `std::string` and support any length.

Reply:
Nice point. I'll leave this for another PR, since here I'm focusing on the queue.